Cargo.lock

```diff
@@ -7944,28 +7912,28 @@ dependencies = [
 
```

Comment:
Oh and `spl-tlv-account-resolution` to v0.5.2

Reply:
I confirmed locally that adding the two bumps I commented on to https://github.com/solana-labs/solana/pull/35165 removes solana-program v1.18.1 from the tree :)